src/reshape.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import {Tensor, sizeOfShape} from './tensor.js';
+
+/**
+ *
```

Comment:
@huningxin Please add the function's description,  likes 'Alter the shape of a tensor to a given new shape. '

Reply:
good catch, fixed in the new commit, PTAL. Thanks.

Reply:
LGTM, thanks.